src/main/java/me/hugeblank/allium/loader/Manifest.java

```diff
@@ -13,14 +15,17 @@ public Manifest(String id, String version, String name, Entrypoint entrypoint) {
         this.entrypoints = entrypoint;
     }
 
+    @LuaWrapped
```

Comment:
You provide access to these methods, but don't provide access to `Script::getManifest`? why's that?

Reply:
Also, IMO the script stuff that can be accessed here (id, name, version) should be accessed in `Script`